CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Add events correlation engine plugin ([#6854](https://github.com/opensearch-project/OpenSearch/issues/6854))
 - Add connectToNodeAsExtension in TransportService ([#6866](https://github.com/opensearch-project/OpenSearch/pull/6866))
 - Adds ExtensionsManager.lookupExtensionSettingsById ([#7466](https://github.com/opensearch-project/OpenSearch/pull/7466))
+- Create NamedRoute to map extension routes to a shortened name ([#6870](https://github.com/opensearch-project/OpenSearch/pull/6870))
```

Comment:
This change needs to be under `Unreleased 2.x` since we are backporting it to `2.x` or you can raise a manual backport PR because the workflow might fail.

Reply:
Moved. That explains why 2 other PRs that I created failed automatic backport. I have created other entries in the past in unreleased 3.0 that are to be released in 2.x. Should those entries be deleted from `Unreleased 3.0`?

These 2 lines:

```
- Add connectToNodeAsExtension in TransportService ([#6866](https://github.com/opensearch-project/OpenSearch/pull/6866))
- Adds ExtensionsManager.lookupExtensionSettingsById ([#7466](https://github.com/opensearch-project/OpenSearch/pull/7466))
```

Reply:
Yes. If any changes are released then it should be a part of the release notes of that specific release. If the above changes are released in any `2.x` release, can you go ahead and remove them from `Unreleased 3.0`.
cc: @kotwanikunal (our CHANGELOG expert)

Reply:
@cwperks can you update the release notes for 2.7 and backport it to 2.x and 2.7?

Reply:
@owaiskazi19 I think its only on main that the CHANGELOG needs to be updated. I opened a PR to cleanup main here: https://github.com/opensearch-project/OpenSearch/pull/7632